### ISSUE TITLE
Hanson rPI-MFC cape not outputting pixels

### ISSUE THREAD

**Issue by robbiet76** [FalconChristmas/fpp#701]
FPP does not appear to output pixel data to the cape, however the panel output is working fine.  According to Alan Hanson, something changed in FPP that will no longer allow output to both.  I know there have been conversations between Alan and Dan, but I haven't seen any related updates in FPP.  I have a cape set up and would be glad to help out with any testing.  Let me know how I can help!

**Comment by dkulp** [comment]
You need to turn on the "Use CPU PWM" setting on the panel output.  Otherwise, the LED panels use the PWM that the pixels need to use.

**Comment by robbiet76** [comment]
I tried that option, but still no luck.  Even when I disable the panel output, still no pixel lights.  I've tried running test mode in FPP as well as running directly from Xlights.  Again, panels work fine from both sources.  Here are some screen shots of settings.  Is there something else I could sendthat might be helpful?

![84B49F57-D9D4-4A31-AD98-1D97954D0130](https://user-images.githubusercontent.com/63306703/78735331-9093c200-7918-11ea-9bd9-783fb45952c9.jpeg)
![B847F718-C397-4DEC-8E6E-56AF96B073EA](https://user-images.githubusercontent.com/63306703/78735333-912c5880-7918-11ea-88aa-9443ff594b70.jpeg)
